models/FanPage.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
+
+const userSchema = new mongoose.Schema(
+  {
+    author: [{
+      type: mongoose.Types.ObjectId,
+      ref: 'User'
+    }],
+    artist: {
+      type: String,
+      required: true,
+    },
+    page_title: {
+      type: String,
+      required: true,
+    },
+    page_bio: {
+        type: String,
+        required: false,
+    },
+    user_tracks: {
+        type: String,
+        required: false,
+    },
+    user_albums: {
+        type: String,
+        required: false,
+    },
+    user_shows: {
+        type: String,
+        required: false,
+    },
+    upvote: [{
+      type: mongoose.Types.ObjectId,
```

Comment:
not sure if it should be 'mongoose.Schema.Types.ObjectId'? double check that

Reply:
According to Stack Overflow they are both the same thing. looks like they are interchangeable.... https://stackoverflow.com/questions/28997636/should-i-use-schema-types-objectid-or-schema-objectid-when-defining-a-mongoose-s